superset/sql_lab.py

```diff
@@ -175,6 +176,7 @@ def handle_error(msg):
             schema=query.schema,
             nullpool=True,
             user_name=user_name,
+            source=sources.get('sql_lab', None),
```

Comment:
So sources.get('sql_lab') will already return a `None` you don't have to explicitly set it here